android-driver/src/main/scala/com/fortysevendeg/mvessel/api/impl/AndroidDatabase.scala

```diff
@@ -0,0 +1,46 @@
+package com.fortysevendeg.mvessel.api.impl
+
+import android.database.sqlite.SQLiteDatabase
+import com.fortysevendeg.mvessel.AndroidDriver
+import com.fortysevendeg.mvessel.api.{CursorProxy, DatabaseProxy, DatabaseProxyFactory}
+import com.fortysevendeg.mvessel.util.ReflectionOps._
+
+class AndroidDatabaseFactory extends DatabaseProxyFactory {
+
+  override def openDatabase(name: String, flags: Int): DatabaseProxy =
+    new AndroidDatabase(SQLiteDatabase.openDatabase(name, null, flags))
```

Comment:
It would be better with the parameter names (more clear) in the `openDataBase` call.